### PR TITLE
Minor  corrections.

### PR DESCRIPTION
**Pull request by chrsmrrs**

### CODE REVIEW

File: 2021/ml4co-competition/index.html [hunk 1]
```diff
@@ -192,15 +192,15 @@ <h2>Challenges</h2>
 				<ul class="features">
 					<li>
 						<h3><a href="#primal-task" class="button">Primal task</a></h3>
-						<p>Produce feasible solutions, in order to minimize the <em>primal integral</em> over time.</p>
+						<p>Produce feasible solutions, minimizing the <em>primal integral</em> over time.</p>
```
Comment (gasse) [inline 2021/ml4co-competition/index.html]: I prefer the previous phrasing. Here it sounds that producing feasible solutions, in whichever way, minimizes the primal integral.

File: 2021/ml4co-competition/index.html [hunk 2]
```diff
@@ -465,15 +465,7 @@ <h2>Metrics</h2>
...
-				Because our evaluation metrics are
-				time-dependent, all evaluations will be run on the same
-				hardware setup, including one CPU core and one GPU card,
-				which will be communicated at the start of the competition.
-				For practical reasons, for each task, a maximum <b>time
-				budget</b> of \(T=15\) minutes is given to process each test instance,
-				after which the environment necessarily terminates.
-				Participants should therefore focus on taking both
-				<b>good and fast decisions</b>.
+
```
Comment (gasse) [inline 2021/ml4co-competition/index.html]: This is not a minor change :P Is this deletion intentional ?

File: 2021/ml4co-competition/index.html [hunk 3]
```diff
@@ -641,11 +633,11 @@ <h2>Metrics</h2>
 				<h2>Datasets</h2>
 			</header>
 			<p>
-				For each individual challenge, the participants will be
+				The participants will be
 				evaluated on <b>three problem benchmarks</b> from diverse application
-				areas. Participants can provide a different
+				areas for each challenge. Participants can provide a different
```
Comment (gasse) [inline 2021/ml4co-competition/index.html]: I like the old sentence better. It is a bit confusing that we have at the same time three tasks and three benchmarks. That's why I tried to be very explicit here: for each individual challenge, the three benchmarks will be used.

File: 2021/ml4co-competition/index.html [hunk 4]
```diff
@@ -654,17 +646,16 @@ <h2>Datasets</h2>
 				(MILP) instances in the standard
 				<a href="https://en.wikipedia.org/wiki/MPS_(format)">MPS file format</a>.
 				Each benchmark is
-				split into <b>a training and a test set</b>, which originate from
+				split into <b>a training and a test set</b>, resulting from
```
Comment (gasse) [inline 2021/ml4co-competition/index.html]: This sounds weird. Samples don't result from a distribution :P

File: 2021/ml4co-competition/index.html [hunk 5]
```diff
@@ -654,17 +646,16 @@ <h2>Datasets</h2>
...
 				the same problem distribution. While the training instances
 				are	made public at the beginning of the competition for
-				participants to train their models, the test instances will
-				be kept hidden for evaluation purposes, and will
-				only be revealed at the end of the competition.
+				participants to train their models. The test instances will
+				be kept hidden for evaluation purposes and will
```
Comment (gasse) [inline 2021/ml4co-competition/index.html:661]: Sounds weird. Either 'While' has to be removed, or the two parts should be in the same sentence.

File: 2021/ml4co-competition/index.html (hunk 1 above)
Comment (chrsmrrs) [reply 2021/ml4co-competition/index.html]: The one I proposed is much more English sounding, it is also semantically equivalent to what you wrote.

Comment (gasse) [reply 2021/ml4co-competition/index.html]: A German and a French debate about the best way to write English. Sounds like the start of a great Joke :P

Comment (chrsmrrs) [reply 2021/ml4co-competition/index.html]: Haha.

**Pull request merged**